app/services/sandbox_seed_data/api_teachers.rb

```diff
@@ -0,0 +1,353 @@
+        :training_period,
+        *training_period_traits.compact,
+        :with_schedule,
+        :for_ect,
+        schedule:,
+        ect_at_school_period: ect_at_school_period_record,
+        school_partnership:,
+        **training_period_data
+      ).tap { |tp| log_training_period(training_period: tp) }
+
+      set_ect_attributes(teacher:)
+      create_teacher_id_change_for(teacher:)
+
+      if Faker::Boolean.boolean(true_ratio: 0.1)
+        mentor_at_school_period_record = mentor_at_school_period(
+          teacher:,
+          school:,
+          school_period:,
+          traits: ect_mentor_traits
+        )
+
+        FactoryBot.create(
+          :training_period,
+          *training_period_traits.compact,
+          :with_schedule,
+          :for_mentor,
+          schedule:,
+          mentor_at_school_period: mentor_at_school_period_record,
+          school_partnership:,
+          **training_period_data
+        ).tap { |tp| log_training_period(training_period: tp) }
+      end
+    end
+
+    def create_mentor_and_optional_ect_training(
+      teacher,
+      school,
+      school_period,
+      schedule,
+      school_partnership,
+      training_period_data,
+      training_period_traits,
+      ect_mentor_traits,
+      ect_specific_traits
+    )
+      mentor_at_school_period_record = mentor_at_school_period(
+        teacher:,
+        school:,
+        school_period:,
+        traits: ect_mentor_traits
+      )
+
+      FactoryBot.create(
+        :training_period,
+        *training_period_traits.compact,
+        :with_schedule,
+        :for_mentor,
+        schedule:,
+        mentor_at_school_period: mentor_at_school_period_record,
+        school_partnership:,
+        **training_period_data
+      ).tap { |tp| log_training_period(training_period: tp) }
+
+      set_mentor_attributes(teacher:)
+      create_teacher_id_change_for(teacher:)
+
+      if Faker::Boolean.boolean(true_ratio: 0.1)
+        ect_at_school_period_record = ect_at_school_period(
+          teacher:,
+          school:,
+          school_period:,
+          traits: ect_mentor_traits + ect_specific_traits
+        )
+
+        FactoryBot.create(
+          :training_period,
+          *training_period_traits.compact,
+          :with_schedule,
+          :for_ect,
+          schedule:,
+          ect_at_school_period: ect_at_school_period_record,
+          school_partnership:,
+          **training_period_data
+        ).tap { |tp| log_training_period(training_period: tp) }
+      end
+    end
+
+    def ect_at_school_period(teacher:, school:, school_period:, traits:)
+      email = Faker::Internet.email(name: ::Teachers::Name.new(teacher).full_name)
+      school_reported_appropriate_body = random_appropriate_body
+
+      FactoryBot.create(
+        :ect_at_school_period,
+        *traits.compact,
+        teacher:,
+        school:,
+        email:,
+        school_reported_appropriate_body:,
+        **school_period
```

Comment:
we need to set the `created_at` to the same time we expect this participant to have been created (maybe started on) as that feeds the created at timestamp in the response, and that can't be after the updated at. Similar in mentors as well